lib/ansible/modules/system/user.py

```diff
@@ -1773,6 +1818,18 @@ def modify_user_usermod(self):
             cmd.append('-s')
             cmd.append(self.shell)
 
+        if self.profile is not None:
```

Comment:
With these changes, it appears the module will report a change every time it is run. Is there a way to inspect the state of the system and determine if these settings need to be applied?

Reply:
I'll try to find a way to do this.

Reply:
I think it will be easier to parse /etc/user_attr instead of parsing the three separate commands.

Reply:
I implemented it via parsing /etc/user_attr